plugins/modules/x509_crl.py

```diff
@@ -51,8 +51,17 @@
               I(revoked_certificates), but can also contain other revoked certificates. If the CRL file
               already exists, all entries from the existing CRL will also be included in the new CRL.
               When using C(update), you might be interested in setting I(ignore_timestamps) to C(true).
+            - The default value is C(generate).
+        type: str
+        # default: generate
+        choices: [ generate, update ]
+        version_added: 2.13.0
+    mode:
+        description:
+            - This parameter is deprecated and will be removed in community.crypto 3.0.0. Use I(crl_mode) instead.
```

Comment:
It looks like this option is the new `crl_mode` option, as defined in line 45, but it's being set at deprecated with a message to use `crl_mode` "instead". Am I misreading something?

Reply:
Yes; basically I renamed `mode` to `crl_mode`, but instead of just adding `mode` as an alias to `crl_mode` I have to keep it as a separate option, since otherwise `mode` as an option would show up from `AnsibleModule`'s `add_file_common_args=True`.

Reply:
How about changing the documentation to:
```suggestion
            - This parameter has been renamed to I(crl_mode). The old name I(mode) is now deprecated and will be removed in community.crypto 3.0.0.
              Simply replace usage of this parameter with I(crl_mode).
```

Reply:
I see my mistake, I completely missed line 59. What had me confused was thinking that 61 was still part of `crl_mode` 🤦 

With that, the old description was fine. The new description is good too; only thing I'd avoid is use of the word "simply" but maybe that's my own feeling only.

Reply:
I removed 'simply', see 6666bafdcd2c90610c1e997d7432405b1248d036.